src/db.c

```diff
@@ -56,6 +56,11 @@ keyStatus expireIfNeeded(serverDb *db, robj *key, int flags);
 int keyIsExpired(serverDb *db, robj *key);
 static void dbSetValue(serverDb *db, robj *key, robj *val, int overwrite, dictEntry *de);
 
+/* Returns which db dbix should be used with kvstore for a given key. */
+static int getKVStoreIndexForKey(sds key) {
+    return server.cluster_enabled ? getKeySlot(key) : 0;
+}
```

Comment:
This is a bit weird. The method seems to belong to kvstore.c like `getIndexForKey` but it again calls `getKeySlot`. So, I guess we need to have it here. Could we do a forward declaration and have the method along side getKeySlot.